app/models/institution.rb

```diff
@@ -0,0 +1,5 @@
+class Institution < ApplicationRecord
+  belongs_to :visit
```

Comment:
Ojo con las relaciones... nunca podés tener dos `belongs_to` en una relación de modelos, ya que esta sentencia es la que determina "a nivel lógico" en qué modelo está la FK [fijate esto](http://requiremind.com/differences-between-has-one-and-belongs-to-in-ruby-on-rails/). De todas formas acá tenemos una relación `1 a N` donde una visita está bien que tenga una FK a institución, pero la inst tiene muchas visitas.